src/views/Login.vue

```diff
@@ -91,11 +93,6 @@ export default defineComponent({
   width: 375px;
 }
 
-img {
```

Comment:
Shouldn't this css be applied to the logo component instead of being entirely deleted?

Reply:
The figure tag itself is having this margin -
![oie_2wGYuPAhcoRA](https://user-images.githubusercontent.com/82817616/151693590-2f1645e9-ce9f-4acc-a0c0-4d076baed9be.png)

and when I am adding more margin and padding to image -
![oie_sh7MC3vJNaQb](https://user-images.githubusercontent.com/82817616/151693636-e8ffe20c-f077-4d41-9b2e-cd618e0f7245.png)

extra space was there that is why I removed margin and padding of image because figure tag was having margin already -
![localhost_8100_login](https://user-images.githubusercontent.com/82817616/151693686-0e04f057-a603-4519-9d4b-b125ccf8aedd.png)

Reply:
where is figure getting its margin from? is it something we set in variables.scss. This could be browser styling that may not be consistent in all browsers. Please check and share findings in comment

Reply:
`figure` tag has default margin of `1em 40px`.

Reply:
https://www.w3schools.com/tags/tag_figure.asp

Reply:
Should I over ride the margin with the old values ?

Reply:
Let's leave it if it's consistent on most browsers.